src/structured-data/handler.js

```diff
@@ -441,8 +405,94 @@ export async function generateSuggestionsData(auditUrl, auditData, context, site
   return { ...auditData };
 }
 
+export async function importTopPages(context) {
+  const { site, finalUrl, log } = context;
+
+  log.info(`Importing top pages for ${finalUrl}`);
+
+  const s3BucketPath = `scrapes/${site.getId()}/`;
+  return {
+    type: 'top-pages',
+    siteId: site.getId(),
+    auditResult: { status: 'preparing', finalUrl },
+    fullAuditRef: s3BucketPath,
+    finalUrl,
+  };
+}
+
+export async function submitForScraping(context) {
+  const {
+    site,
+    dataAccess,
+    log,
+    finalUrl,
+  } = context;
+  const { SiteTopPage } = dataAccess;
+  const topPages = await SiteTopPage.allBySiteIdAndSourceAndGeo(site.getId(), 'ahrefs', 'global');
```

Comment:
Optional: Here we are using SiteTopPages.allBySiteIdAndSourceAndGeo method, but elsewhere in code we are using getTopPagesForSiteId method for top pages (which in turn calls same method). You may want to use same method for sake of consistency.